rubin_sim/maf/metrics/snNSNMetric.py

```diff
@@ -236,6 +234,13 @@ def run(self, dataSlice, slicePoint):
         metricVal : `np.recarray`
             ['nSN', 'zlim'] at this point on the sky
         """
+        # remove or keep DD runs depending on fieldType to process
+
+        DDobs = np.in1d(dataSlice[self.noteCol], self.DD_list)
+        if self.fieldType == 'DD':
+            dataSlice = dataSlice[DDobs]
+        else:
+            dataSlice = dataSlice[~DDobs]
```

Comment:
I understand why you have included this here. 
Typically we take care of this in the calling script (removing or allowing observations based on the "note" field is a sqlconstraint kind of thing for the MetricBundle).
If you need us to leave this here, so that your copy of the code can track ours, we can do that. 
I would note that the list of DD fields can potentially change, so you would need to modify the calling script and metric configuration in that case.

Reply:
The list od DD fields is an input parameter of the metric. So it can be changed when necessary. I have added these lines because the templates of gamma_WFD.hdf5 are not valid for all DDFs. So I thought that it would be better to be sure that the processing is done either for WFD or for DDFs.